packages/core/stories/patterns/contact-details/contact-details.stories.tsx

```diff
@@ -0,0 +1,288 @@
+import persona from "../../assets/avatar.png";
+
+export default {
+  title: "Patterns/Contact Details",
+} as Meta;
+
+const basicContact = {
+  primary: "Jane Doe",
+  secondary: "This is a long secondary title to show that the text will wrap",
+  metadata: [
+    {
+      label: "Role",
+      value: "Analyst",
+      icon: <UserBadgeIcon aria-hidden />,
+    },
+    {
+      label: "Location",
+      value: "London, GBR",
+      icon: <LocationIcon aria-hidden />,
+    },
+    {
+      label: "Office",
+      value: "+44 2077 431102",
+      icon: <CallIcon aria-hidden />,
+    },
+    {
+      label: "Email",
+      value: "email@company.com",
+      icon: <MessageIcon aria-hidden />,
+    },
+  ],
+};
+
+export const ContactDetails = ({ avatarSrc = persona }) => {
+  return (
+    <StackLayout
+      direction={"row"}
+      gap={2}
+      style={{
+        width: "400px",
+      }}
+    >
+      <Avatar
+        src={avatarSrc as string}
+        aria-label={basicContact.primary}
+        fallbackIcon={<UserBadgeIcon />}
+        size={2}
+      />
+      <StackLayout direction={"column"} gap={1}>
+        <Text styleAs="h3">{basicContact.primary}</Text>
+        <Text styleAs="h4">{basicContact.secondary}</Text>
+        <StackLayout direction={"row"} gap={2}>
+          <StackLayout gap={1} direction={"column"}>
+            {basicContact.metadata.map((metadata) => {
+              return (
+                <Text variant="secondary" key={metadata.label}>
+                  {metadata.label}
+                </Text>
+              );
+            })}
+          </StackLayout>
+          <StackLayout gap={1} direction={"column"}>
+            {basicContact.metadata.map((metadata) => {
+              return <Text key={metadata.label}>{metadata.value}</Text>;
+            })}
+          </StackLayout>
+        </StackLayout>
+      </StackLayout>
+    </StackLayout>
+  );
+};
+
+export const Horizontal = () => {
+  return (
+    <StackLayout direction={"row"} gap={2} style={{ width: "400px" }}>
+      <Avatar
+        src={persona as string}
+        aria-label={basicContact.primary}
+        fallbackIcon={<UserBadgeIcon />}
+        size={2}
+      />
+      <StackLayout direction={"column"} gap={1}>
+        <Text styleAs="h3">{basicContact.primary}</Text>
+        <Text styleAs="h4">{basicContact.secondary}</Text>
+        <FlowLayout gap={1}>
+          {basicContact.metadata.map((metadata) => {
+            return (
+              <FlexLayout
+                key={metadata.label}
+                align="center"
+                justify="center"
+                gap={1}
+              >
+                {metadata.icon}
+                <Text key={metadata.label} aria-label={metadata.label}>
+                  {metadata.value}
+                </Text>
+              </FlexLayout>
+            );
+          })}
+        </FlowLayout>
+      </StackLayout>
+    </StackLayout>
+  );
+};
+
+export const WithIcon = () => {
```

Comment:
Should this change the label to icons instead of the avatar?

Reply:
The name should be updated to make it clear, e.g. With Avatar Icon or Vertical - with Icon